src/components/MintDurationTiers/MintDurationTiers.tsx

```diff
@@ -0,0 +1,114 @@
+import { BodyXs, Flex, H5, Stack, VStack } from "@threshold-network/components"
+import { FC } from "react"
+import { getRangeSign } from "../../utils/getRangeSign"
+import { getDurationByNumberOfConfirmations } from "../../utils/tBTC"
+import { StackProps } from "@threshold-network/components"
+import { ComponentPropsWithoutRef } from "react"
+import { RangeOperatorType, CurrencyType } from "../../types"
+import { useThreshold } from "../../contexts/ThresholdContext"
+
+type BaseProps = ComponentPropsWithoutRef<"li"> & StackProps
+type MintDurationTiersItemProps = {
+  amount: number
+  currency: CurrencyType
+  rangeOperator: RangeOperatorType
+}
+
+interface MintDurationTiersProps extends BaseProps {
+  items: MintDurationTiersItemProps[]
+}
+
+const MintDurationTiers: FC<MintDurationTiersProps> = ({
+  items,
+  ...restProps
+}) => {
+  const {
+    tbtc: {
+      minimumNumberOfConfirmationsNeeded: getNumberOfConfirmationsByAmount,
+    },
+  } = useThreshold()
+
+  return (
+    <Stack
+      direction={{
+        base: "column",
+        sm: "row",
+      }}
+      spacing={{
+        base: 3,
+        md: 6,
+      }}
+      {...restProps}
+    >
+      {items.map(({ amount, rangeOperator, currency }, index) => {
+        const correctedAmount =
+          amount + (rangeOperator.includes("greater") ? 0.01 : -0.01)
+        // The amount is corrected by adding or subtracting 0.01 to the given amount
+        // depending on the range operator. This is done to avoid floating-point errors
+        // when comparing BigNumber values.
```

Comment:
Comment exceeds 80 characters margin

```suggestion
        // The amount is corrected by adding or subtracting 0.01 to the given
        // amount depending on the range operator. This is done to avoid
        // floating-point errors when comparing BigNumber values.
```

Reply:
Ref commit: 7f8e2357